src/macaron/__main__.py

```diff
@@ -301,14 +290,15 @@ def main() -> None:
     logger.info("The logs will be stored in debug.log")
 
     # Set Macaron's global configuration.
+    # The path to provenance expectation files will be updated if
+    # set through analyze sun-command.
```

Comment:
```suggestion
    # set through analyze sub-command.
```
Unless it's meant to be `sun-command` :smile:

Reply:
See commit [c09c5a3](https://github.com/oracle-samples/macaron/pull/192/commits/c09c5a3379ef0733cd926c0b767480d77809a220).